README.md

```diff
@@ -13,6 +13,8 @@
 
 ## Libraries
 
+- [alex-snezhko/hopper](https://github.com/alex-snezhko/hopper) - An HTTP microframework for the Grain programming language built on top of Wagi
```

Comment:
```suggestion
- [Hopper](https://github.com/alex-snezhko/hopper) - An HTTP microframework for the Grain programming language built on top of Wagi
```

Reply:
for that I was following the convention used `[deislabs/wagi-fileserver](https://github.com/deislabs/wagi-fileserver) — A static file server for Wagi` in the wagi fileserver link, so if we change it here we should probably change it fro that one.

Reply:
Different PR